Documentation/heisenbug.rst

```diff
@@ -10,67 +10,84 @@ It also has 2 GPGPUs (NVIDIA GeForce RTX 3090), that can be used to run the GPU
 The RTX 3090 belongs to a consumer kind of graphics cards and thus does not perform well with double precision. 
 Therefore, it is preferable to compile SeisSol with single precision.
 
-A module integrating all libraries relevant for compiling SeisSol with CUDA and SYCL is available on heisenbug.
+CPU and GPU seissol modules are available on heisenbug. They also integrate all libraries relevant for manually compiling another version of SeisSol.
 It can be discovered at startup after adding the following to ``~/.bashrc``:
 
 .. code-block:: bash
 
-    module use /import/exception-dump/ulrich/spack/modules/linux-debian11-zen2
+    module use /import/exception-dump/ulrich/spack/modules/linux-debian12-zen2
 
-It is then loaded with:
+E.g. the GPU module can be loaded with:
 
 .. code-block:: bash
 
-    # load the (first in the list) seissol-env module compiled with cuda support
-    module load $(module avail seissol-env/*-cuda-* | awk '/seissol-env/ {print $1}')
+    # load the (first in the list) seissol module compiled with cuda support
+    module load $(module avail seissol/*-cuda-* | awk '/seissol/ {print $1}')
 
-This module has been compiled based on the main branch of https://github.com/SeisSol/seissol-spack-aid with the command:
+These modules have been compiled based on the main branch of https://github.com/SeisSol/seissol-spack-aid with the command:
 
 .. code-block:: bash
 
-    spack install -j 40 --fresh seissol-env +cuda %gcc@10
-    spack module tcl refresh $(spack find -d --format "{name}{/hash:5}" seissol-env +cuda)
-
-Install YATeTo GPU backends (i.e., GemmForge and ChainForge) as 
-shown :ref:`here <gemmforge_installation>`.
-
-Then clone SeisSol with:
-
-.. code-block:: bash
-
-    git clone https://github.com/SeisSol/SeisSol.git
-    cd SeisSol
-    git submodule update --init --recursive
-
-
-To compile the GPU version of SeisSol on heisenbug, use the following cmake options 
-
-.. code-block:: bash
-
-    -DDEVICE_ARCH=sm_86 -DHOST_ARCH=hsw -DDEVICE_BACKEND=cuda -DPRECISION=single -DHIPSYCL_CUDA_PATH=$CUDA_HOME
+    spack compiler find
+    # deactivate conda module (if any)
+    conda deactivate
+    # python has an externally-managed-environment on heisenbug
+    # We therefore need to create an environment, e.g. with:
+    python3 -m venv /import/exception-dump/ulrich/python-venv
+    export PATH=/import/exception-dump/ulrich/python-venv/bin:$PATH
```

Comment:
Is this path accessible to everyone when they run it?

Reply:
no, it is just an example, there the "e.g."